language/compiler/ir-to-bytecode/syntax/src/spec_language_ast.rs

```diff
@@ -13,7 +13,8 @@ pub enum StorageLocation {
     Formal(String),
     /// A resource of type `type_` stored in global storage at `address`
     GlobalResource {
-        type_: StructName,
+        type_: QualifiedStructIdent,
```

Comment:
This will break compilation of spec_translator.rs. There is a larger PR coming up which refactors this code heavily for type checking. You could make a minimal fix to get past rust compilation errors and ignores Boogie compilation (which isn't run by default by tests).

Reply:
I just fixed spec_translator locally--let me update this PR and see what you think. I think it should keep Boogie compilation working as well.